forge-gui/res/cardsfolder/upcoming/snarlfang_vermin.txt

```diff
@@ -0,0 +1,12 @@
+Name:Snarlfang Vermin
+ManaCost:B
+Types:Creature Rat
+PT:2/1
+T:Mode$ DamageDone | CombatDamage$ True | ValidSource$ Card.Self | ValidTarget$ Creature | TriggerZones$ Battlefield | Execute$ TrigSuspect | TriggerDescription$ Whenever CARDNAME deals combat damage to a creature, if that creature is still on the battlefield, suspect that creature. It perpetually gains this ability.
+SVar:TrigSuspect:DB$ AlterAttribute | Defined$ TriggeredTargetLKICopy | RememberAltered$ True | Attributes$ Suspected | SubAbility$ DBAnimate
```

Comment:
What happens if the damaged creature dies? I feel like this line should check that the Defined is still on the battlefield.

Reply:
Like this ?

Reply:
condition typo

Reply:
also RememberAltered doesn't work like you want

Reply:
> also RememberAltered doesn't work like you want

What do you mean ? Is this not correct ?

Reply:
not if it's already suspected

Reply:
Is the current version correct ?

Reply:
So there shoudn't be a condition on the animate. I thought the oracle meant that the suspected creature gains this ability ?

Reply:
no, I said **different condition**
there's only one here which is the "inverening if"

Reply:
Ok thanks for explaining